SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosTaskBuilder.java

```diff
@@ -119,8 +119,9 @@ public SingularityTask buildTask(SingularityOfferHolder offerHolder, List<Resour
     } else if (MesosUtils.getDisk(offerHolder.getCurrentResources(), Optional.absent()) >= 1.0) {
```

Comment:
Just realized, we should be passing the requiredRole here instead of an absent

Reply:
Hmm, so if the task actually requests disk, we do pass along the requiredRole (in the `if()` just above this line). For this case, though, the task hasn't requested any disk resources, but we just claim some to enable disk reporting. In this case, we don't care if the role actually matches, do we?